cupy/typing/_generic_alias.py

```diff
@@ -18,6 +18,8 @@
 
 if sys.version_info >= (3, 9):
     from types import GenericAlias
+elif numpy.lib.NumpyVersion(numpy.__version__) >= '1.23.0':
+    from numpy._typing import _GenericAlias as GenericAlias
```

Comment:
Seems that `_GenericAlias` is moved to `numpy._typing` since numpy 1.23. I wonder why we first see this failure here, though.

Reply:
ah, I found it in cupy.win.cuda113, which is not required in master test.